src/applications/personalization/account/containers/AccountApp.jsx

```diff
@@ -16,7 +16,6 @@ class AccountApp extends React.Component {
     return (
       <div>
         <RequiredLoginView
-          authRequired={1}
```

Comment:
this prop is no longer supported by RequiredLoginView. It was removed here https://github.com/department-of-veterans-affairs/vets-website/commit/96d073c40b8282a24979b01713fa6050d1dede8b